test/libsolidity/syntaxTests/conversion/not_allowed_conversion_to_int_array_pointer2.sol

```diff
@@ -0,0 +1,10 @@
+ contract C {
+        uint[] a;
+        uint[] b;
+        function f() public view {
+            uint[] storage c = a;
+            uint[] memory d = b;
+            d = uint[](c);
```

Comment:
Can you check that we have tests that show that `c = d` and `c = uint[](d)` does not work?

Reply:
will do soon.

Reply:
added a test case  for these two situation in the folder "dataLocations".